src/viur/core/email.py

```diff
@@ -449,3 +452,57 @@ def check_sib_quota() -> None:
             entity["latest_warning_for"] = None
 
         db.Put(entity)
+
+class EmailTransportMailjet(EmailTransport):
+    @staticmethod
+    def splitAddress(address: str) -> dict[str, str]:
+        """
+            Splits a Name/Address Pair into a dict,
+            i.e. "Max Musterman <mm@example.com>" into
+            {"name": "Max Mustermann", "email": "mm@example.com"}
+            :param address: Name/Address pair
+            :return: split dict
+        """
+        posLt = address.rfind("<")
+        posGt = address.rfind(">")
+        if -1 < posLt < posGt:
+            email = address[posLt + 1:posGt]
+            sname = address.replace(f"<{email}>", "", 1).strip()
+            return {"name": sname, "email": email}
+        else:
+            return {"email": address}
```

Comment:
This method, apart from the docstring, looks the same as `EmailTransportSendInBlue.splitAddress`. To avoid duplicate code I think it would be better to move the method to module level or into utils where both classes can use the same method.